lib/beaker/hypervisor/vmpooler.rb

```diff
@@ -19,6 +19,18 @@ def initialize(vmpooler_hosts, options)
       @options = options
       @logger = options[:logger]
       @hosts = vmpooler_hosts
+      @credentials = load_credentials(@options[:dot_fog])
```

Comment:
Let it try to load the file? Have `load_credentials` successfully catch exceptions and print a line in the log that it failed to load `.fog` and is continuing without credentials.